x/staking/keeper/keeper.go

```diff
@@ -119,7 +120,8 @@ func NewKeeper(
 			),
 			codec.CollValue[types.Redelegation](cdc),
 		),
-		UnbondingIndex: collections.NewMap(sb, types.UnbondingIndexKey, "unbonding_index", collections.Uint64Key, collections.BytesValue),
+		UnbondingIndex:       collections.NewMap(sb, types.UnbondingIndexKey, "unbonding_index", collections.Uint64Key, collections.BytesValue),
+		UnbondingDelegations: collections.NewMap(sb, types.UnbondingDelegationKey, "unbonding_delegation", collections.PairKeyCodec(sdk.LengthPrefixedAddressKey(sdk.AccAddressKey), sdk.ValAddressKey), collections.BytesValue), // nolint: staticcheck // sdk.LengthPrefixedAddressKey is needed to retain state compatibility
```

Comment:
the first part of a bytes key is by default length prefixed, the second part does not need to be length prefixed since the remainder of a buffer key already is the key itself, but in the SDK we decided to also length prefix the secondary key. 


So the key should be: `collections.PairKeyCodec(sdk.AccAddressKey, sdk.ValAddressKey)`